src/dev/build/tasks/fleet/bundle_packages.ts

```diff
@@ -23,6 +23,7 @@ interface FleetPackage {
   name: string;
   version: string;
   forceAlignStackVersion?: boolean;
+  allowSyncToPrerelease?: boolean;
```

Comment:
Is it needed to be added here the definition of the field or in any other file? Currently, it is not going to be used here. It will be used as part of the jobs to update the version of the packages.